tfe/testing.go

```diff
@@ -44,3 +44,19 @@ func skipIfEnterprise(t *testing.T) {
 		t.Skip("Skipping test for a feature unavailable in Terraform Enterprise. Set 'ENABLE_TFE=0' to run.")
 	}
 }
+
+func isAcceptanceTest() bool {
+	return os.Getenv("TF_ACC") == "1"
+}
+
+// Most tests relyo terraform-plugin-sdk/helper/resource.Test to run.
```

Comment:
😄 Don't bother changing, I'm pushing your commit for CI anyway

```suggestion
// Most tests rely on terraform-plugin-sdk/helper/resource.Test to run.
```